Visualization/CombineBlastTables.R

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env Rscript
+
+# CombineBlastTables.R
+# Copyright Jackson M. Tsuji, 2018
+# Neufeld Research Group
+# See required R packages below.
+
+#####################################################
+## User variables: #################################
+RUN_COMMAND_LINE <- TRUE # If selected, all user input here is ignored, and terminal-based input is expected instead.
+
+# Set other user variables here
+if (RUN_COMMAND_LINE == FALSE) {
+  setwd("/Users/JTsuji/Downloads") # your working directory where files are stored
+  input_filenames <- c("subject1.csv", "subject2.csv", "subject3.csv") # NOTE: filename minux extesion will become the subject_name!
+  output_filename <- "combined_blast_tables.csv"
+}
+#####################################################
+
+#####################################################
+## Load required packages: ##########################
+library(getopt)
+library(tools)
+library(plyr)
+suppressMessages(library(dplyr))
+#####################################################
+
+# Hard-coded variables
+header_names <- c("qseqid", "sseqid", "pident", "evalue", "qcovhsp", "bitscore")
+
+
+# Function to assign command line input to variables, or throw help message
+parse_command_line_input <- function() {
+  
+  inputArgs <- commandArgs(trailingOnly = TRUE)
+  if (length(inputArgs) < 2) {
+    cat("CombineBlastTables.R: Combines BLAST tables and makes new column for sample ID based on the filenames.\n")
+    cat("Copyright Neufeld Lab, 2018\n")
```

Comment:
I'm not sure how the copyrights are going to work. Is this your standard header?

Reply:
@LeeBergstrand  Yes, this is my standard header. I can change to "Copyright Lee Bergstrand and Jackson M. Tsuji, 2018" -- would that be more generic for now? That's what I've added elsewhere in the script.